sort/dutchsationalslagsort_test.go

```diff
@@ -0,0 +1,34 @@
+package sort
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestDutchNationalFlagSort(t *testing.T) {
```

Comment:
There is already a [testing framework](https://github.com/TheAlgorithms/Go/blob/4f43cc154fccf98cccd49c1b1fdba21cba8e4a5f/sort/sorts_test.go#L11) in this package to test sorting algorithms matching the sort function signature, you only need to provide it with your function and test on that.


All you need is this:

```go
func TestDutchNationalFlag(t *testing.T) {
	testFramework(t, sort.DutchNationalFlag[int])
}
```

I would also prefer to keep this test closer to all the other tests in the `sort_test.go` file - since that is the convention we have used for this package but moving this to that file is a non-blocker.